.travis.yml

```diff
@@ -1,5 +1,3 @@
-dist: xenial
```

Comment:
This is the default for Travis builds, setting it here isn't necessary. (azavea/django-amazon-ses#60)